.gitignore

```diff
@@ -1 +1,2 @@
-node_modulesgit
\ No newline at end of file
+node_modules
+coverage
```

Comment:
**Nitpick**
Please, use [editorConfig](http://editorconfig.org/) to remove `EOF`